packages/react-image/src/components/Image/useImage.ts

```diff
@@ -8,13 +8,12 @@ export const imageShorthandProps: Array<keyof ImageSlots> = ['root'];
  * Given user props, returns state and render function for an Image.
  */
 export const useImage = (props: ImageProps, ref: React.Ref<HTMLImageElement>): ImageState => {
-  const { bordered, fit, fluid, circular, rounded } = props;
+  const { bordered: bordered, fit, block, shape = 'square' } = props;
   const state: ImageState = {
-    bordered,
+    bordered: bordered,
```

Comment:
Was there any issue with having only as `bordered,`?

Reply:
No that is a typo left over from me experimenting with naming it something different.  I was thinking maybe it should be bordered: 'subtle' | 'strong' or something like that, but decided that was too much change just before beta.  I'll fix that.